libraries/vendors/compile-altera.sh

```diff
@@ -513,7 +513,63 @@ if [ $STOPCOMPILING -eq 0 ] && [ "$COMPILE_ARRIA" == "TRUE" ] && [ $SKIP_LARGE_F
 
 	GHDLCompilePackages
 fi
+# compile Cyclone library
+if [ $STOPCOMPILING -eq 0 ] && [ "$COMPILE_CYCLONE" == "TRUE" ]; then
+	Library="cyclone"
+	Files=(
+		cyclone_atoms.vhd
+		cyclone_components.vhd
+	)
 
+	# append absolute source path
+	SourceFiles=()
+	for File in ${Files[@]}; do
+		#Don't put nonexisting files.
+		if [ -f "$SourceDirectory/$File" ]; then
+			SourceFiles+=("$SourceDirectory/$File")
+		fi
```

Comment:
Shouldn't the script generate a message in case `VERBOSE` is enabled that a file was skipped, because it was missing?

Reply:
Now that you mention it, there is negation of it... SUPPRESS_WARNINGS=0. I actually tried to find the verbose switch...
I'm not really fluent with bash, but I can use it. Anyway this is starting to look something like function for addind  $SourceFiles.